### PR TITLE
Update cannot-expand-nested-contact-groups-unknown-shown.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated another resolution to remove the lost members

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/OfficeDocs-Support/issues/1718

File: Outlook/Client/contacts/cannot-expand-nested-contact-groups-unknown-shown.md [hunk 1]
```diff
@@ -30,9 +30,22 @@ In Microsoft Outlook, you try expand a contact group in a public folder that con
...
-## Resolution
+## Resolution 
+  
+### Method 1
 
 To resolve this issue, right-click the Contact folder in the Public Folder, and then select **Properties**. On the Outlook **Address Book** tab, select the **Show this folder as an email address book** option.
+  
+### Method 2
+  
+1) Open the group, Select **Update now**. If there are members that cannot be found, outlook will prompt you to choose between the three below options
```
Comment (JohanFreelancer9) [inline Outlook/Client/contacts/cannot-expand-nested-contact-groups-unknown-shown.md:41]:
```suggestion
1) Open the group and select **Update now**. If there are members that cannot be found, Outlook will prompt you to choose between the following three options:
```

File: Outlook/Client/contacts/cannot-expand-nested-contact-groups-unknown-shown.md [hunk 2]
```diff
@@ -30,9 +30,22 @@ In Microsoft Outlook, you try expand a contact group in a public folder that con
...
+  
+  - Remove the lost members.
+  - Try to repair  the lost members and remove only those that cannot be resolved.
```
Comment (JohanFreelancer9) [inline Outlook/Client/contacts/cannot-expand-nested-contact-groups-unknown-shown.md:44]:
```suggestion
  - Try to repair the lost members and remove only those that cannot be resolved.
```

File: Outlook/Client/contacts/cannot-expand-nested-contact-groups-unknown-shown.md [hunk 3]
```diff
@@ -30,9 +30,22 @@ In Microsoft Outlook, you try expand a contact group in a public folder that con
...
+  - Cancel this operation and do not change any members.
+ 
+2) Select, **Try to repair  the lost members and remove only those that cannot be resolved** and click **Ok**.
```
Comment (JohanFreelancer9) [inline Outlook/Client/contacts/cannot-expand-nested-contact-groups-unknown-shown.md:47]:
```suggestion
2) Choose **Try to repair the lost members and remove only those that cannot be resolved** and select **Ok**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @msbemba, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**